app/views/archive_faqs/_navigation.html.erb

```diff
@@ -0,0 +1,22 @@
+<h3 class="landmark heading"><%= t(".landmark") %></h3>
+<ul class="navigation actions">
+  <% if policy(ArchiveFaq).translation_access? %>
+    <li>
+      <%= span_if_current t(".archive_faq"), archive_faqs_path %>
+    </li>
+  <% end %>
```

Comment:
I'm pretty sure you can leave this check out. If you can see this navigation partial, you should be able to access archive_faqs_path.